docs/src/appendix/snforge-library/get_call_trace.md

```diff
@@ -0,0 +1,109 @@
+# `get_call_trace`
+
+```rust
+fn get_call_trace() -> CallTrace;
+```
+
+(For whole structure definition, please refer to [`snforge-std` source](https://github.com/foundry-rs/starknet-foundry/tree/v0.16.0/snforge_std))
+
+Gets latest call trace of the test, including the last call made to a contract.
+
+## Example call trace
+```cairo
+use snforge_std::trace::{CallTrace, CallEntryPoint, CallType, EntryPointType, get_call_trace};
+...
+
+let ctrace = CallTrace {
+    entry_point: CallEntryPoint {
+        entry_point_type: EntryPointType::External,
+        entry_point_selector: test_selector(),
+        calldata: array![],
+        contract_address: test_address(),
+        caller_address: 0.try_into().unwrap(),
+        call_type: CallType::Call,
+    },
```

Comment:
We should mention that in a full trace this top entry point is fixed and symbolises call made to the test code (since we are in "invoke"), as it's not obvious